modules/serverless-sinkbinding-yaml.adoc

```diff
@@ -11,6 +11,23 @@ This guide describes the steps required to create, manage, and delete a SinkBind
 
 * You have Knative Serving and Eventing installed.
 
+[NOTE]
+====
+The following procedure requires you to create YAML files.
+
+If you change the names of the YAML files from those used in the examples, you must ensure that you also update the corresponding CLI commands.
+====
+
+[IMPORTANT]
```

Comment:
I think that this information should be a step in a procedure instead of just a note in the prereqs. If you're trying to avoid putting this step that requires admin permissions in a procedure that you're expecting users without elevated permissions to complete, then I think that it should go in another assembly and the need to have it done should be a line item in these modules. (I'm not sure which assembly - I don't see a configuring topic that's targeted for admins after a quick scan of the serverless book.)

That said, I think that it's ok as-is as long as you keep it in mind as a candidate for a future configuring topic.

Reply:
Will do. I had planned some time ago to make an admin guide but got busy with other stuff and ended up closing the PR because there was too much structure change for it to be worth all the rebasing and redirects. I will revisit this effort after 4.6 release :slightly_smiling_face: thank you as always for the good input!

Reply:
Absolutely! I'm happy to help. :)